### PR TITLE
UNI-458 shows user that data in file details is a subset

### PR DESCRIPTION
**Pull request by andrewmalta13**
Modifies FileDetails.jsx to make it clear that the data shown in file details is a subset of the actual data from the file. 

The text/style used to illuminate this point is flexible and should be updated in a new JIRA ticket if needed.

Please Review:
@marionleborgne

### CODE REVIEW

**Comment by marionleborgne** [comment]
👍 @andrewmalta13. The text and implementation looks good. However, the text is not aligned and is hard to notice all the way at the bottom.  How about you place it on in the right space above the CSV table? I.e. in the `TextField` component with `File Size` and `Number of rows` (except it does not have a title and it's italicized) Your text is good. Just remove the dot at the end.
screen shot 2016-05-27 at 1 42 24 am

**Comment by andrewmalta13** [comment]
how does that look? @marionleborgne

**Comment by andrewmalta13** [comment]
updated wrapping issue @marionleborgne

**Comment by marionleborgne** [comment]
@andrewmalta13 👍 I can merge once you merge master into this branch. Here is the merge conflict resolution: https://gist.github.com/marionleborgne/125305f33faabcd16e3f62b2c06a3c40

**Comment by andrewmalta13** [comment]
@marionleborgne addressed your comment

File: unicorn/app/browser/components/FileDetails.jsx [hunk 1]
```diff
@@ -85,14 +85,21 @@ const STYLES = {
...
+  },
   supportedFormats: {
     cursor: 'pointer',
     textDecoration: 'underline'
   }
 };
 
 // The errors that will trigger a (see supported formats) in the error message
-const SUPPORTED_FORMAT_ERRORS = ['The CSV file does not have any valid data']
+const SUPPORTED_FORMAT_ERRORS = ['The CSV file does not have any valid data',
```
Comment (marionleborgne) [inline unicorn/app/browser/components/FileDetails.jsx]: @andrewmalta13 I think you should remove this and append the `(see supported formats)` message at the end of all error messages. I notified celeste about that, the QA plan will be updated.

**Comment by marionleborgne** [comment]
👍

**Pull request merged**